backend/src/API/shoutoutAPI.ts

```diff
@@ -42,3 +42,17 @@ export const hideShoutout = async (
   if (!shoutout) throw new NotFoundError(`Shoutout with uuid: ${uuid} does not exist!`);
   await ShoutoutsDao.updateShoutout({ ...shoutout, hidden: hide });
 };
+
+export const deleteShoutout = async (uuid: string, user: IdolMember): Promise<void> => {
+  const shoutout = await ShoutoutsDao.getShoutout(uuid);
+  if (!shoutout) {
+    throw new NotFoundError(`No shoutout with id '${uuid}' found.`);
+  }
+  const isLeadOrAdmin = await PermissionsManager.isLeadOrAdmin(user);
+  if (!isLeadOrAdmin || shoutout.giver.email !== user.email) {
```

Comment:
This should be `&&`, since leads/admins or the shoutout giver can delete the shoutout.

Reply:
This means all members cannot delete any shoutouts.

Reply:
It should be `!(isLeadOrAdmin || shoutout.giver.email !== user.email)` or `!isLeadOrAdmin && shoutout.giver.email !== user.email`

Reply:
That makes sense, I will change it.